app/javascript/components/top-corner-controller.jsx

```diff
@@ -26,7 +35,13 @@ const Row = styled.div`
   margin-right: 100px;
 `;
 
-const TopCorner = ({ showWeather, cityName, loading, error, location }) => {
+export const TopCorner = ({
+  showWeather,
+  cityName,
+  loading,
+  error,
+  location,
+}) => {
```

Comment:
Should this be added in its own refactor commit?

Reply:
I don't think so. This is a very minor "refactor" and is probably not worth a separate commit. But maybe a mentor has another opinion.